src/backend/ManagementHub.Storage/Migrations/20251231093148_AddTournamentRosters.cs

```diff
@@ -0,0 +1,2901 @@
+﻿using System;
```

Comment:
Something is wrong with this migration. Please remove it and recreate it correctly by calling `dotnet ef migrations new` in the working directory set to `src/backend/ManagementHub.Storage` - it should be using .NET 8 SDK.
Make sure to remove the Designer file associated and revert changes to the DB context snapshot.